readme.md

```diff
@@ -67,5 +65,7 @@ kaliber: {
 
 ![](https://media.giphy.com/media/SUp0ZNb0pmL3G65I2k/giphy.gif)
 
+## Build steps before releasing a new version
+Because this library is used in both the node runtime and webpack environment, the library is compiled using kaliberjs/pack (https://github.com/kaliberjs/pack). When releasing a new version, do not forgot to execute `yarn build` before running `yarn publish`
```

Comment:
```suggestion
## Before releasing a new version
Because this library is used in both the node runtime and webpack environment, the library is precompiled using kaliberjs/pack (https://github.com/kaliberjs/pack). When releasing a new version, do not forgot to execute `yarn build` before running `yarn publish`. This should be done because the yarn publish command uses your localfiles to upload to the npm registy.
```